crates/autopilot/src/run.rs

```diff
@@ -586,12 +592,11 @@ pub async fn run(args: Arguments) {
         .await
         .expect("failed to perform initial solvable orders update");
 
-    let now = Arc::new(RwLock::new(Instant::now()));
-    let liveness = Liveness {
+    let liveness = Arc::new(Liveness {
```

Comment:
nit: Liveness could have a constructor and set last_auction_time itself (given it's not really variable and not needed to be overridden in tests)